c/example01.c

```diff
@@ -35,10 +35,15 @@ int main(void){
   const char *ak = getenv("OSC_ACCESS_KEY");
   const char *sk = getenv("OSC_SECRET_KEY");
 
+  if (ak == NULL || sk == NULL) {
+    printf("ak or sk not declared");
```

Comment:
`fprintf(stderr, "OSC_ACCESS_KEY or OSC_SECRET_KEY environement variable not set\n")`
fprintf so you can use on stderr instead of stdout
and don't forget the '\n' please